### PR TITLE
added models

### PR DESCRIPTION
**Pull request by Bekhzod7799**

### CODE REVIEW

File: TestHotel/TestHotel.DataAccess/Model/Role.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class Role
+    {
+        public int RoleID { get; set; }
+        public string RoleTitle { get; set; }
+        public string RoleDescription { get; set; }
+
+        public Employee Employee { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Role.cs]: employee listda bo'lishi kerak edi

File: TestHotel/TestHotel.DataAccess/Model/Employee.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        public string LastName { get; set; }
+        public string BirthdayData { get; set; }
+        public enum Gender { get; set; }
+        public string PhoneNumber { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public int Salary { get; set; }
+
+        public Hotel Hotel { get; set; }
+        public List<Roles> Role { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Employee.cs]: Role listda bo'lishi kerak emas edi

File: TestHotel/TestHotel.DataAccess/Model/RoomType.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class RoomType
+    {
+        public int RoomTypesID { get; set: }
+        public enum RoomTypes { get; set; }
+        public int RoomPrice { get; set; }
+        public string RoomImg { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/RoomType.cs]: RoomImg ni RoomImages ga o'zgartirish kerak

File: TestHotel/TestHotel.DataAccess/Model/RoomType.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class RoomType
+    {
+        public int RoomTypesID { get; set: }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/RoomType.cs]: RoomTypesID dan "s" ni olip tashlash kerak

File: TestHotel/TestHotel.DataAccess/Model/Room.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class Room
+    {
+        public int RoomNumber { get; set; }
+        public int RoomTypes { get; set; }
+        public enum RoomTypesID { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Room.cs]: RoomTypesID dan "s"ni olish va enumni int qilish kerak edi

File: TestHotel/TestHotel.DataAccess/Model/Room.cs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class Room
+    {
+        public int RoomNumber { get; set; }
+        public int RoomTypes { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Room.cs]: RoomTypes ni enum qilish kerak

File: TestHotel/TestHotel.DataAccess/Model/Hotel.cs [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    internal class Hotel
+    {
+        public int HotelID { get; set; }
+        public string HotelName { get; set; }
+        public string Address { get; set; }
+        public string City { get; set; }
+        public string Country { get; set; }
+        public int NumberRooms { get; set; }
+        public string PhoneNumber { get; set; }
+        public int StarRating { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Hotel.cs:25]: manmcha buni keremas diyudila

**Review by MirazamTUT — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Model/RoomType.cs [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+        public enum RoomTypes { get; set; }
+        public int RoomPrice { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/RoomType.cs]: int ni double qilish kerak masmidi?

**Review by MirazamTUT — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Model/Booking.cs [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class Booking
+    {
+        public int BookingId { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Booking.cs:11]: HotelID qo'shmabsiz diyagrammada bor ekan

File: TestHotel/TestHotel.DataAccess/Model/Employee.cs [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestHotel.DataAccess.Model
+{
+    internal class Employee
+    {
+        public int EmployeeID { get; set; }
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Model/Employee.cs]: HotelID qopketibdi chunki diagrammada bor edi

**Review by MirazamTUT — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Model/Booking.cs (hunk 9 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Booking.cs:11]: Ok, HotelCode Hotel Id ga alishtirildi.

File: TestHotel/TestHotel.DataAccess/Model/Employee.cs (hunk 10 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Employee.cs]: Ok, done.

File: TestHotel/TestHotel.DataAccess/Model/Employee.cs (hunk 2 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Employee.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/Role.cs (hunk 1 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Role.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/Room.cs (hunk 6 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Room.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/Room.cs (hunk 5 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Room.cs]: ok done

File: TestHotel/TestHotel.DataAccess/Model/RoomType.cs (hunk 3 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/RoomType.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/RoomType.cs (hunk 8 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/RoomType.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/RoomType.cs (hunk 4 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/RoomType.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/Hotel.cs [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+namespace TestHotel.DataAccess.Model
+{
+    internal class Hotel
+    {
+        public int HotelId { get; set; }
+        public string HotelName { get; set; }
+        public string Address { get; set; }
+        public string City { get; set; }
+        public string Country { get; set; }
+        public int NumberRooms { get; set; }
```
Comment (isfandiyoro5) [inline TestHotel/TestHotel.DataAccess/Model/Hotel.cs]: NumberOfRooms

File: TestHotel/TestHotel.DataAccess/Model/Booking.cs [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+        public int HotelId { get; set; }
+        public int GuestId { get; set; }
+        public int RoomNumber { get; set; }
+        public DateTime BookingDate { get; set; }
+        public DateTime BookingTime { get; set; }
+        public DateTime ArrivalDate { get; set; }
+        public DateTime DepartureDate { get; set; }
+        public int NumberAdults { get; set; }
+        public int NumberChildren { get; set; }
+        //public string SpecialRequest { get; set; } (alohida table qilinadi)
```
Comment (isfandiyoro5) [inline TestHotel/TestHotel.DataAccess/Model/Booking.cs]: bu request ochiriladi

File: TestHotel/TestHotel.DataAccess/Model/Employee.cs [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+namespace TestHotel.DataAccess.Model
+{
+    internal class Employee
+    {
+        public int EmployeeID{ get; set; }
+        public int HotelId { get; set; }
+        public int RoleID { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public DateTime BirthdayDate { get; set; }
```
Comment (isfandiyoro5) [inline TestHotel/TestHotel.DataAccess/Model/Employee.cs]: BirthDate bolishi kk

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Model/Hotel.cs (hunk 11 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Hotel.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/Employee.cs (hunk 13 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Employee.cs]: ok, done

File: TestHotel/TestHotel.DataAccess/Model/Booking.cs (hunk 12 above)
Comment (Bekhzod7799) [reply TestHotel/TestHotel.DataAccess/Model/Booking.cs]: ok, done

**Review by isfandiyoro5 — APPROVED** [review]

**Review by MirazamTUT — APPROVED** [review]

**Pull request merged**